config.def.h

```
@@ -384,6 +390,7 @@ static Shortcut shortcuts[] = {
 	{ TERMMOD,              XK_space,       keyboard_select, { 0 } },
 	{ TERMMOD,              XK_F,           searchforward,   { 0 } },
 	{ TERMMOD,              XK_B,           searchbackward,  { 0 } },
+	{ ControlMask,          XK_i,           keyboard_flash,  { 0 } },
```

You can't use Ctrl+i as the default shortcut here because apps can use it too. For example, Vim uses that shortcut for its jump list. The default mask should be TERMMOD (Ctrl-Shift) and users can then change it as they like in their config.h file.

---

The keysym should be `XK_I` when the mask contains ShiftMask, otherwise it won't work.  This has fooled me a couple of times, because in the dwm configuration it does not have to be in capital letters in the corresponding case.

---

done